backend/main.py

```
@@ -110,4 +108,111 @@ async def analyze_glaucoma(file: UploadFile = File(...)):
+                "data": {
+                    "has_glaucoma": is_glaucoma,
+                    "confidence": confidence,
+                    "cup_to_disc_ratio": cup_to_disc_ratio,
+                    "image_base64": img_base64
+                }
+            }) + "\n"
+
+            if os.path.exists(tmp_path):
+                os.remove(tmp_path)
+
+        except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)